gensim/summarization/bm25.py

```diff
@@ -152,14 +154,68 @@ def get_scores(self, document, average_idf):
         return scores
 
 
-def get_bm25_weights(corpus):
+def _get_scores(bm25, document, average_idf):
+    """Helper function for retrieving bm25 scores of given `document` in parallel
+    in relation to every item in corpus.
+
+    Parameters
+    ----------
+    bm25 : BM25 object
+        BM25 object fitted on the corpus where documents are retrieved.
+    document : list of str
+        Document to be scored.
+    average_idf : float
+        Average idf in corpus.
+
+    Returns
+    -------
+    list of float
+        BM25 scores.
+
+    """
+    scores = []
+    for index in xrange(bm25.corpus_size):
+        score = bm25.get_score(document, index, average_idf)
+        scores.append(score)
+    return scores
+
+
+def _effective_n_jobs(n_jobs):
```

Comment:
Can you move this function to `gensim.utils` and rename it to `effective_n_jobs` (looks useful for later usage).

Reply:
@menshikh-iv thanks. I have fixed them.